go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/opencontainers/runc v0.1.1 // indirect
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
+	github.com/pkg/errors v0.8.1
```

Comment:
Why? :o

Reply:
Don't remember adding that.

Reply:
Can happen by accident when you do `errors.New()` and if you have previously installed pkg.errors on your system (some other project), it'll come up as a suggestion rather than the default errors package. Et voila, new dependecy :D